src/plugins/vis_type_timeseries/public/application/components/vis_types/timeseries/vis.js

```diff
@@ -235,11 +234,15 @@ class TimeseriesVisualization extends Component {
             legend={Boolean(model.show_legend)}
             legendPosition={model.legend_position}
             tooltipMode={model.tooltip_mode}
-            xAxisLabel={getAxisLabelString(interval)}
             xAxisFormatter={this.xAxisFormatter(interval)}
             annotations={this.prepareAnnotations()}
             syncColors={syncColors}
             palettesService={palettesService}
+            interval={interval}
+            shouldDropLastBucket={Boolean(
```

Comment:
I think  user should see that message if `DropLastBucket` is `false` =)

![image](https://user-images.githubusercontent.com/20072247/114998964-1a6e5780-9eaa-11eb-88dd-e8dff0bfb5cb.png)